include/freerdp/channels/rdpgfx.h

```diff
@@ -97,9 +97,10 @@ typedef struct
 #define RDPGFX_CAPVERSION_104 0x000A0400 /** [MS-RDPEGFX] 2.2.3.7 */
 #define RDPGFX_CAPVERSION_105 0x000A0502 /** [MS-RDPEGFX] 2.2.3.8 */
 #define RDPGFX_CAPVERSION_106 0x000A0601 /** [MS-RDPEGFX] 2.2.3.9 */
-#define RDPGFX_CAPVERSION_107 0x000A0701 /** [MS-RDPEGFX] 2.2.3.10 */
+#define RDPGFX_CAPVERSION_106_ERR 0x000A0600 /** [MS-RDPEGFX]-180912-errata] */
+#define RDPGFX_CAPVERSION_107 0x000A0700     /** [MS-RDPEGFX] 2.2.3.10 */
 
```

Comment:
In the spec RDPGFX_CAPVERSION_107 is 0x000A0701 not 0x000A0700

Reply:
true, fixing...